app-frontend/src/app/core/services/gridLayer.service.js

```diff
@@ -0,0 +1,187 @@
+         * @returns {promise} API response for scene grid
+         */
+        requestGrid(coords, params) {
+            let url = `/api/scene-grid/${coords.z}/${coords.x}/${coords.y}/`;
+            return this.$http.get(url, {params: params});
+        }
+
+        /** Create grid layer given a set of filter parameters
+         *
+         * @param {object} params filter parameters for scene grid
+         *
+         * @returns {GridLayer} grid layer used to control display of scene summary for tiles
+         */
+        createNewGridLayer(params) {
+            let self = this;
+
+            delete params.bbox;
+
+            let GridLayer = L.GridLayer.extend({
+
+                requestGrid: self.requestGrid.bind(self),
+
+                params: params,
+
+                /** Function called when parameters have been updated to force a redraw
+                 *
+                 * @param {object} updateParams updated filter parameters for layer
+                 *
+                 * @returns {null} null
+                 */
+                updateParams: function (updateParams) {
+                    delete params.bbox;
+                    this.params = updateParams;
+                    this.redraw();
+                },
+
+                /** Function to calculate shade of grid cell based on number of contained scenes
+                 *
+                 * @param {number} number count of scenes, used to determine shade
+                 * @returns {string} rgba formatted string for scene grid color
+                 */
+                getColor: (number) => {
+                    if (number === 0) {
+                        return 'rgba(53,60,88,0)';
+                    } else if (number > 0 && number < 5) {
+                        return 'rgba(53,60,88,.15)';
+                    } else if (number >= 5 && number < 30) {
+                        return 'rgba(53,60,88,.35)';
+                    } else if (number >= 30 && number < 60) {
+                        return 'rgba(53,60,88,.60)';
+                    }
+                    return 'rgba(53,60,88,.70)';
+                },
+
+                /** Function used to create tile for custom grid layer
+                 *
+                 * @param {object} coords upper left coordinates of tile
+                 * @param {function} done callback used to show tile generation finished
+                 * @return {div} tile container
+                 */
+                createTile: function (coords, done) {
+                    let tile = L.DomUtil.create('div', 'leaflet-tile');
+
+                    let stage = new Konva.Stage({
+                        container: tile,
+                        width: 256,
+                        height: 256
+                    });
+
+                    let text = new Konva.Text({
+                        x: 0,
+                        y: 0,
+                        align: 'center',
+                        fontFamily: 'Titillium Web',
```

Comment:
Doesn't look like this font is being used when the text is actually drawn for some reason. It's a bit hard to tell though.